product-matrix.json

```diff
@@ -15,50 +15,26 @@
     },
     {
       "channel": "stable",
-      "comments": "Android Studio 3.6 RC 1, IntelliJ 2019.2.4",
+      "comments": "Android Studio 3.6, IntelliJ 2019.2.4",
       "name": "Android Studio 3.6",
       "version": "3.6",
       "ideaProduct": "android-studio",
-      "ideaVersion": "192.6071332",
+      "ideaVersion": "192.6200805",
       "dartPluginVersion": "192.7761",
-      "sinceBuild": "192.6262",
+      "sinceBuild": "192.7142",
       "untilBuild": "192.*"
     },
     {
       "channel": "stable",
-      "comments": "IntelliJ 2019.3",
+      "comments": "IntelliJ 2019.3.3, Android Studio 4.0 beta",
       "name": "IntelliJ 2019.3",
-      "version": "2019.3",
+      "version": "4.0",
       "isTestTarget": "true",
-      "ideaProduct": "ideaIC",
-      "ideaVersion": "2019.3",
-      "dartPluginVersion": "193.5432",
+      "ideaProduct": "android-studio",
+      "ideaVersion": "193.6220182",
+      "dartPluginVersion": "193.6494.35",
       "sinceBuild": "193.3519.25",
       "untilBuild": "193.*"
-    },
-    {
-      "channel": "dev",
-      "comments": "Android Studio 4.0 Canary 8",
-      "name": "Android Studio 4.0",
-      "version": "4.0",
-      "ideaProduct": "android-studio",
-      "ideaVersion": "193.6107147",
-      "dartPluginVersion": "193.5731",
-      "sinceBuild": "193.5233",
-      "untilBuild": "193.*",
-      "filesToSkip": ["src/io/flutter/editor/IntellijColorPickerProvider.java"]
-    },
-    {
-      "channel": "stable",
-      "comments": "IntelliJ 2020.1 EAP",
```

Comment:
I'd definitely like to try and fix this (removing for now sounds good). Can you file a bug with the current problems w/ supporting EAP?

Reply:
https://github.com/flutter/flutter-intellij/issues/4404